### PR TITLE
Update inventoryCtrl.js (pop-up fixin')

### PR DESCRIPTION
**Pull request by tjwds**
Names with spaces, yeah yeah yeah

Before: 
capture decran 2014-05-05 a 10 02 41 pm

After: 
capture decran 2014-05-06 a 12 32 25 pm

### CODE REVIEW

**Comment by tjwds** [comment]
Note:  the stuff in habitrpg-shared still needs to be fixed, but this changes saddling, feeding prompts, and hatching to be correct.  The "[egg] really likes it!" pop-up is still broken.

**Comment by tjwds** [comment]
So… uh… as far as I can tell, this is compatible with the update @paglias just made, but something's going to have to change if we want the actual names of the pets + food to be localized.

File: public/js/controllers/inventoryCtrl.js [hunk 1]
```diff
@@ -78,7 +78,10 @@ habitrpg.controller("InventoryCtrl", ['$rootScope', '$scope', '$window', 'User',
     }
 
     $scope.hatch = function(egg, potion){
-      if (!$window.confirm(window.env.t('hatchAPot', {potion: potion.key, egg: egg.key}))) return;
+		var eggName = Content.eggs[egg.key].text;
```
Comment (paglias) [inline public/js/controllers/inventoryCtrl.js:81]: Content.eggs[egg.key].text()

File: public/js/controllers/inventoryCtrl.js [hunk 2]
```diff
@@ -78,7 +78,10 @@ habitrpg.controller("InventoryCtrl", ['$rootScope', '$scope', '$window', 'User',
...
+		var potName = Content.hatchingPotions[potion.key].text;
```
Comment (paglias) [inline public/js/controllers/inventoryCtrl.js:82]: var potName = Content.hatchingPotions[potion.key].text();

File: public/js/controllers/inventoryCtrl.js [hunk 3]
```diff
@@ -95,8 +98,8 @@ habitrpg.controller("InventoryCtrl", ['$rootScope', '$scope', '$window', 'User',
         User.user.ops.purchase({params:{type:type,key:item.key}});
     }
 
-    $scope.choosePet = function(egg, potion){
-      var petDisplayName = potion + " " + egg,
+    $scope.choosePet = function(egg, potion, name){
+      var petDisplayName = Content.hatchingPotions[potion].text + " " + Content.eggs[egg].text,
```
Comment (paglias) [inline public/js/controllers/inventoryCtrl.js:102]: var petDisplayName = env.t('petName', {potion: Content.hatchingPotions[potion].text(), egg: Content.eggs[egg].text()});

**Comment by paglias** [comment]
@tjwds commented with the changes you need to make to have it working, won't be able to test it so please test it but it should work

**Comment by tjwds** [comment]
I'm having serious issues getting a local copy to run, but I'll get back to you as soon as I can get it to work.

**Comment by paglias** [comment]
@tjwds I did this https://github.com/HabitRPG/habitrpg/commit/df8eb107cda3cd78f59735cd63547242fbb81620

Thanks anyway!

**Pull request closed without merging**